cmd/sops/subcommand/exec/exec.go

```diff
@@ -134,6 +135,14 @@ func ExecWithEnv(opts ExecOpts) error {
 
```

Comment:
At the top of this function, please check whether if `runtime.GOOS == "windows" && opts.SameProcess`, and either return a nice `error` in that case, or print a warning and set `opts.SameProcess = false` (similar to https://github.com/getsops/sops/blob/08476d1bed45d7b95ec7482fdaf47baf65bb258a/cmd/sops/subcommand/exec/exec.go#L63-L66).

Right now the caller can only see from the log output that something went wrong and the command wasn't run at all; the exit code will be 0 since no error is reported.